tests/docker-compose-testing.yml

```diff
@@ -14,7 +14,7 @@ services:
     #   dockerfile: docker/Dockerfile
     #   target: resolver
     # CAUTION: Change IMAGE_VERSION to local in docker-compose.env if building your own image in section below
-    image: cheqd/did-resolver:staging-latest
+    image: ${IMAGE_NAME:-cheqd/did-resolver}:staging-latest
```

Comment:
This doesn't look right; it's probably a typo.
Should be `${IMAGE_NAME}:staging-latest`, right @rblaine95 ?

Reply:
`${IMAGE_NAME}` is the environment variable that has `cheqd/did-resolver` or `rblaine95/did-resolver`, etc
`:-cheqd/did-resolver` is the default if the environment variable doesn't exist.

So:
`export IMAGE_NAME=foo/bar/baz` will result in `image: foo/bar/baz:staging-latest`
`unset IMAGE_NAME` will result in `cheqd/did-resolver:staging-latest`

Reply:
https://docs.docker.com/compose/how-tos/environment-variables/variable-interpolation/#interpolation-syntax

Reply:
In the context of these actions, `IMAGE_NAME` is `${{ github.repository }}`

So, in the DIDx fork for this PR, that resolves to `didx-xyz/cheqd-did-resolver`.

Adding this was required to get the pipeline to mostly run in the fork.